### PR TITLE
Diagram Improvements in Book

### PR DESCRIPTION
**Pull request by Water261**
# Change Summary
- Replace use cases diagram with a mermaid flowchart
- Change out git branch ASCII art with mermaid git graph

Fixes #
- N/A

Checklist
- [x] This PR contains no AI generated code
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

**Comment by Water261** [comment]
image
image

^ Photos so you don't need to spin up mdbook

File: book/src/introduction_to_kanidm.md [hunk 1]
```diff
@@ -26,8 +26,29 @@ Kanidm solves this problem by acting as a central authority of accounts in your
...
+	device<-- Trusts OAuth -->website;
+	website<-- Reads User Data -->kanidm;
+	device-- OAuth -->kanidm;
+
+	device-- SSH -->ssh;
+	ssh<-- Retrieve SSH Keys -->kanidm;
+
+	device-- Login to -->wifi;
+	wifi-- RADIUS Credentials -->radius;
+	radius<-- Verifies Credentials -->kanidm;
```
Comment (yaleman) [inline book/src/introduction_to_kanidm.md:49]: Minor nit, Kanidm doesn't reach back out for things 😄 

```suggestion
	device-- Trusts OAuth -->website;
	website-- Reads User Data -->kanidm;
	device-- OAuth -->kanidm;

	device-- SSH -->ssh;
	ssh-- Retrieve SSH Keys -->kanidm;

	device-- Login to -->wifi;
	wifi-- RADIUS Credentials -->radius;
	radius-- Verifies Credentials -->kanidm;
```

File: book/src/support.md [hunk 2]
```diff
@@ -41,6 +41,51 @@ the pre-release version.
               |        |   tag v1.2.1
               |        \- tag v1.2.0
               \- tag v1.2.0-pre
+``` -->
```
Comment (yaleman) [inline book/src/support.md:44]: Fine to remove the whole block

**Review by yaleman — CHANGES_REQUESTED** [review]
Minor nits, but thanks

**Review by yaleman — APPROVED** [review]

**Pull request merged**